notebooks/post_DL3_analysis.ipynb

```diff
@@ -0,0 +1,2633 @@
+{
```

Comment:
<p>What are the two set of points?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
I missed specifying the `sed_type` in the `plot()`. Fixing it.